todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksViewModel.kt

```diff
@@ -47,32 +27,76 @@ import com.example.android.architecture.blueprints.todoapp.util.EDIT_RESULT_OK
  * getter method.
  */
 class TasksViewModel(
-        context: Application,
-        private val tasksRepository: TasksRepository
+    context: Application,
+    private val tasksRepository: TasksRepository
 ) : AndroidViewModel(context) {
```

Comment:
does this need to be an AndroidViewModel?

Reply:
It does, I'm accessing lots of resources